src/utils/waitForComponentToPaint.js

```diff
@@ -0,0 +1,11 @@
+// if the state changes when mounting a component, use this
+import { act } from 'react-dom/test-utils';
+
+const waitForComponentToPaint = async (wrapper) => {
```

Comment:
better move this file to a new folder in `__test__` called `utilities`, because this is a function used only in unit tests

Reply:
I tried that, npm test executes this and fails, not sure how to make jest ignore folders when testing, will check out again

Reply:
I am sure that there must be a way. Did you try to google it?

Reply:
Adding testpathignorepatters in package.json 
`  "jest": {
    "testPathIgnorePatterns": [
      "<rootDir>/__test__/helpers/"
    ]
  }`
  Does not seem to do the trick - https://jestjs.io/docs/en/configuration#testpathignorepatterns-arraystring

Reply:
then in that case maybe put this file in a new folder in `src/utils/tests` to make it explicit that it is for the tests